### ISSUE TITLE
Benchmarks?

### ISSUE THREAD

**Issue by pombredanne** [mischasan/aho-corasick#13]
Did you ever compare the perfs against https://github.com/pfac-lib/pfac ? That would be quite interesting...

**Comment by mischasan** [comment]
No, just fgrep, which seemed to be a reasonably-tuned algorithm.
I don't have much time right now. If you do that test, let me know.
